test/index.js

```diff
@@ -40,46 +48,38 @@ describe('HtmlWebpackNewRelicPlugin', () => {
           },
         );
 
-        expect(file).include('licenseKey');
-        expect(file).include('applicationID');
+        for (const [optionName, optionValue] of Object.entries(testPluginOptions)) {
+          // 'license' is the only option that doesn't match what is used in the script
+          const scriptOptionName = (optionName === 'license' ? 'licenseKey' : optionName);
```

Comment:
**Reviewer Note:** Since this PR already has a breaking change, should we use this as an opportunity to rename `license` to `licenseKey` for consistency?

Reply:
Thanks.  You are correct, and I think I will go ahead and update this.

Reply:
@feanil: I would have done this, but upstream merged an earlier version of this PR before I was able to make the change (https://github.com/victorrodrigues/html-webpack-new-relic-plugin/pull/5), and enough time has passed and I haven't even been able to test/use this yet, so I am going to unfortunately skip this change.  Too bad.